content/networking/ip-optimization-strategies/index.md

```diff
@@ -0,0 +1,95 @@
+# Optimizing IP Address Utilization 
+
+Containerized environments are growing in scale at a rapid pace, thanks to application modernization. This means that more and more worker nodes and pods are being deployed.
+
+The [Amazon VPC CNI](../vpc-cni/) plugin assigns each pod an IP address from the VPC CIDR(s), which can consume a substantial number of IP addresses from your VPCs depending on your workload.
```

Comment:
I recommend adding following otherwise one always would think of : _Why did Amazon make this decision then ?_ 

which essentially makes the Pod a first-class citizen in the VPC. This approach provides full visibility of the Pod addresses with tools such as [VPC Flow Logs](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html) and other monitoring solutions. On the other hand depending on your workload type this can cause a substantial number of IP addresses to be consumed by the pods.